envoy/config/metrics/v2/stats.proto

```diff
@@ -155,6 +155,9 @@ message StatsdSink {
     // Envoy will connect to this cluster to flush statistics.
     string tcp_cluster_name = 2;
   }
+  // [#not-implemented-hide:] Optional custom prefix for StatsdSink. If
+  // specified, this will override the default prefix.
```

Comment:
Can you potentially given an example of how this would be used and what change it will make in the emitted metrics?

Reply:
Updated.

Reply:
Can you specify what the default is if not specified? "envoy" ?

Reply:
Done!